### DISCUSSION TITLE
Forward syslog directly to splunk enterprise from an nginx-plus ingress controller

### DISCUSSION THREAD

**Discussion by nguyenphu123** [nginx/nginx#198] [Q&A]
Hi, I am currently working on an nginx plus as ingress controller for my kubernetes and using sc4s to forward logs to splunk enterprise. However I notice that sc4s does not forward all of logs include the approtect WAF and DoS. Does the WAF and DoS require special setup to forward logs? I tried with syslog-ng [https://github.com/nginxinc/kubernetes-ingress/blob/v3.6.2/examples/ingress-resources/app-protect-do...](https://github.com/nginxinc/kubernetes-ingress/blob/v3.6.2/examples/ingress-resources/app-protect-dos/README.md) like this example but the logs is not showing on splunk enterprise.

Thanks.

**Comment by mrajagopal** [comment]
What is your configuration for WAF's log destination?
As an example:
```
      logDest: "syslog:server=syslog-svc.default:514"
```
The NGINX Ingress Controller documentation has a section titled [NGINX App Protect WAF Logs](https://docs.nginx.com/nginx-ingress-controller/installation/integrations/app-protect-waf/configuration/#waf-logs)

Can you share the relevant config?

**Comment by Maryna-f5** [comment]
Correct project for this discussion is here https://github.com/nginxinc/kubernetes-ingress
